BunqSdk.Tests/Model/Generated/Endpoint/TabUsageSingleTest.cs

```diff
@@ -16,21 +16,21 @@ public class TabUsageSingleTest : BunqSdkTestBase
         /// <summary>
         /// Config values
         /// </summary>
-        private const string TAB_FIELD_DESCRIPTION = "Pay the tab for Java test please.";
-        private const string FIELD_STATUS_OPEN = "OPEN";
-        private const string AMOUNT_EUR = "10.00";
-        private const string FIELD_CURRENCY = "EUR";
-        private const string TAB_ITEM_FIELD_DESCRIPTION = "Super expensive java tea";
-        private const string FIELD_STATUS_WAITING = "WAITING_FOR_PAYMENT";
+        private const string TabDescription = "Pay the tab for Java test please.";
+        private const string StatusOpen = "OPEN";
```

Comment:
`TabStatusOpen`

Reply:
@OGKevin it should be TabUsageSingleStatusOpen following the style you describe above regarding the group, right? *naming is hard*

Reply:
@antao Naming is indeed hard! 😁 but you will get used to it!

Yes `TabUsageSingleStatusOpen` sounds good 👍